scripts/tests/scripts_regression_tests.py

```diff
@@ -164,6 +166,8 @@ def test_lib_doctests(self):
                     else:
                         print("{} has no doctests".format(filepath))
 
+        self.assertTrue(get_model() == "e3sm")
```

Comment:
How will this work for cesm?

Reply:
@jedwards4b , sorry, this line was mistakenly left in the code. I just removed it.